docs/src/basic_concepts/record_fields.md

```diff
@@ -0,0 +1,31 @@
+## Record organization
+The format uses various fields to organize and distinguish **records**
+and **record sets**[^1]:
+`name`, `uas_id` and `record_number`.
+
+### Records
+**Records** are used to store data both in **detection** as well as **tracking**.
+Each record must have a unique `record_number` inside **detection** or **tracking**,
+meaning that the same `record_number` can appear in both without causing conflict.
```

Comment:
![image](https://github.com/COURAGEOUS-isf/format/assets/16354904/76768c40-7762-41f1-b4e9-39f75f1c0f2c)
Bold text is way too distracting, use it sparingly. Don't use it on detection/tracking here.